src/index.js

```diff
@@ -1,52 +1,26 @@
-import './styles.scss';
-import reload from './assets/refresh.png';
-import enter from './assets/enter.png';
-import addList from './addList.js';
+import './styles.css';
+import Task from './task.js';
 import List from './List.js';
-import Store from './store.js';
-import dummyList from './dynamicObj.js';
 
-const heading = document.getElementById('reloadImg');
-heading.src = reload;
-
-document.getElementById('addBtn').innerHTML = `
-<img src=${enter} alt="enter" />
-`;
+const task = new Task();
 
 document.getElementById('form').addEventListener('submit', (e) => {
   e.preventDefault();
-  const input = document.getElementById('input').value;
-  if (input === '') {
-    addList.showAlert();
-  } else {
-    const list = new List(input);
-    addList.addList(list);
-    Store.addList(list);
-    addList.clearFields();
-  }
+  const description = document.getElementById('input').value;
+  const list = new List(description);
+  task.addList(list);
```

Comment:
- [ ] Currently, the problem with deleting items from your todo app persists, kindly check out this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) to guide  you on how the functionality works